### PR TITLE
Glossary section

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
We discussed in one of PRs that the part of spec clean up we need to align terminology in the document. This is a foundation for this clean up. Any concerns with `Glossary` section introduction?

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/pull/231.html" title="Last updated on Jan 19, 2019, 12:48 AM UTC (d5d75ab)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/trace-context/231/8f6f492...d5d75ab.html" title="Last updated on Jan 19, 2019, 12:48 AM UTC (d5d75ab)">Diff</a>

### CODE REVIEW

File: spec/10-overview.md [hunk 1]
```diff
@@ -1,16 +1,40 @@
 # Overview
 
-Trace context consists of a set of properties crucial for event correlation in distributed applications. Trace context positions requests in a distributed trace. If you look at a distributed trace as a graph of correlated requests - a single request can be part of multiple graphs. For example, let's say component `A` calls component `B` which in turn calls `C`. One distributed tracing system may see component `A` calling component `B`. Another may see all three components communicating. So the call from `B` to `C` will carry trace context about service `A` as well as information about its direct communication upstream to `B`.
+Trace context consists of a set of properties crucial for event
+correlation in distributed applications. Trace context positions
+requests in a <a>distributed trace</a>. If you look at a <a>distributed
```
Comment (SergeyKanzhelev) [inline spec/10-overview.md]: this is the only not a new-line modification - adding `<a></a>` twice here.

Comment (yurishkuro) [reply spec/10-overview.md]: what is the purpose of `<a></a>` without href?

Comment (SergeyKanzhelev) [reply spec/10-overview.md]: It's a way ReSpec marks mentions of definitions. https://github.com/w3c/respec/wiki/ReSpec-Editor's-Guide#definitions-and-linking. I didn't find a better way to do it that will be markdown friendly and easy to use

Comment (AloisReitbauer) [reply spec/10-overview.md]: This PR changes way more than just a glossary. The changes conflict with a lot of other PRs. Please make this more granular

**Comment by danielkhan** [comment]
The changes to spec/20-HTTP_HEADER_FORMAT.md collide with the ongoing wording changes in https://github.com/w3c/trace-context/pull/227. Yet, I think we can merge and then replicate the changes in the new version of the doc. I can do that.

**Review by mtwo — APPROVED** [review]
Made some small grammar changes to the new text in index.html

Approved, though we need to ensure that this does't cause conflicts with Daniel's changes

**Comment by SergeyKanzhelev** [comment]
this is approved for many days now. No issues raised. Can this be merged?

**Review by bogdandrutu — APPROVED** [review]

**Review by AloisReitbauer — APPROVED** [review]

**Pull request merged**